SI/Sem.05/task07.cpp

```diff
@@ -0,0 +1,32 @@
+﻿#include <iostream>
+
+void PrintMatrix(int arr[][3], int rows) {
+    for (int row = 0; row < rows; row++)
+    {
+        for (int col = 0; col < 3; col++)
+        {
+            std::cout << arr[row][col] << " ";
+        }
+        std::cout << std::endl;
+    }
+}
+
+void transp(int arr[][3], int newArr[][3], int rows) {
+
+    for (int row = 0; row < rows; row++)
+    {
+        for (int col = 0; col < 3; col++)
+        {
+            newArr[col][row] = arr[row][col];
+        }
+    }
+}
```

Comment:
The functions should work with matrixes NxN, currently it is working with Nx3. Solution: parameters of the functions should be int**